Movie/Movie/Module/MovieList/View/Cell/AboutMovieTableViewCell.swift

```diff
@@ -52,16 +52,16 @@ final class AboutMovieTableViewCell: UITableViewCell {
 
     // MARK: - Public Methods
 
-    func configure(movie: Movie, imageService: ImageServiceProtocol?) {
+    func configure(movie: Movie, viewModel: MovieListViewModelProtocol?) {
         movieNameLabel.text = movie.title
         ratingLabel.text = String(movie.voteAverage)
-        fetchCoverMovieImage(movie: movie, imageService: imageService)
+        fetchCoverMovieImage(movie: movie, viewModel: viewModel)
     }
 
     // MARK: - Private Methods
 
-    private func fetchCoverMovieImage(movie: Movie, imageService: ImageServiceProtocol?) {
-        imageService?.fetchImage(imagePath: movie.posterPath) { [weak self] result in
+    private func fetchCoverMovieImage(movie: Movie, viewModel: MovieListViewModelProtocol?) {
+        viewModel?.imageService.fetchImage(imagePath: movie.posterPath) { [weak self] result in
```

Comment:
```suggestion
        viewModel?.fetchImage(imagePath: movie.posterPath) { [weak self] result in
```
обращаемся через метод вьюмодели к сервису